### ISSUE TITLE
genesis.json

### ISSUE THREAD

**Issue by lrettig** [ewasm/testnet#31]
child of #20

**Comment by jwasinger** [comment]
What is this?

**Comment by jwasinger** [comment]
Okay.  I looked into this somewhat recently in BA.  I opted not to use puppeth because I didn't see a use for it at the time.  It was only potentially going to come into play if we were to use the same faucet as Ropsten.  The metamask faucet appears to work fine with cpp-ethereum/hera.

As far as configuration goes, I haven't figured out how to generate a key pair for cpp ethereum node P2P Id.  We need this in order to hardcode the peerlist for each running testnet node in the cluster.

@chfast if you have any pointers on this ^ it would be helpful for me.
